source/jormungandr/jormungandr/instance.py

```diff
@@ -393,7 +393,7 @@ def update_property(self, response):
         """
         update the property of an instance from a response if the metadatas field if present
         """
-        if response.HasField(b"metadatas") and response.publication_date != self.publication_date:
+        if response.HasField(str("metadatas")) and response.publication_date != self.publication_date:
```

Comment:
`str()` ensure that we have the desired string type no matter what version of python we are using.
`unicode_literals` is enable so every quoted string is a `unicode` object, Protobuf doesn't always like them, that's kinda strange. With `str()` we have a ascii string aka `bytes` in python2 and a unicode string in python 3.